### PR TITLE
Handle beta leap

### PR DESCRIPTION
**Pull request by OleksandrOrlov**
Handle BETA popup on Leap.

- Related ticket: https://progress.opensuse.org/issues/97490
- Verification runs:  
    - Leap (validating BETA text and confirm): https://openqa.opensuse.org/tests/1922284
    - Leap (just confirm): https://openqa.opensuse.org/tests/1922480
    - TW (no BETA, to show that it is not broken): https://openqa.opensuse.org/tests/1922479
    - SLE (just confirm): https://openqa.suse.de/tests/7154394
    - SLE (validate tex and confirm): https://openqa.suse.de/tests/7154395

### CODE REVIEW

File: schedule/yast/opensuse/installer_extended/installer_extended.yaml [hunk 1]
```diff
@@ -29,3 +30,8 @@ conditional_schedule:
     BACKEND:
       qemu:
         - installation/language_keyboard/switch_keyboard_layout
+  access_beta_distribution:
+    BETA:
+      1:
+        - installation/validate_beta_message
+        - installation/access_beta_distribution
```
Comment (jknphy) [inline schedule/yast/opensuse/installer_extended/installer_extended.yaml:37]: I guess we will need to set beta_text here too, right?

**Review by jknphy — COMMENTED** [review]
LGTM besides that minor thing.

File: schedule/yast/opensuse/installer_extended/installer_extended.yaml (hunk 1 above)
Comment (OleksandrOrlov) [reply schedule/yast/opensuse/installer_extended/installer_extended.yaml:37]: Fixed and started a new VR. Good catch:)

Comment (OleksandrOrlov) [reply schedule/yast/opensuse/installer_extended/installer_extended.yaml:37]: It passed through the step: https://openqa.opensuse.org/tests/1926714

**Pull request merged**